http_calls.py

```diff
@@ -73,6 +74,14 @@ def GET(self, url, headers={}, args=None):
 
         return self.read(req)
 
+    def Header(self, h, v):
+
+        global g_headers
+        g_headers[h] = v
```

Comment:
Why all? We are talking about AT here, which should be transparent. IMHO cleaning header by header in SetUp or SuiteSetUp is right and transparent.